base/formats.go

```diff
@@ -197,3 +198,28 @@ func TrimBase64Mime(b64 string) string {
 	splitB64 := strings.Split(b64, ",")
 	return splitB64[len(splitB64)-1]
 }
+
+// return the extension of the file from the base64 string, in the "jpeg" , "png" format, check with magic number
+func GetBase64FileExtensionSlow(b64 string) string {
```

Comment:
Did we use this function somewhere? If not, let's remove it and rename `GetBase64FileExtensionFast` to `GetBase64FileExtension`

Reply:
Fixed.